components/slider/index.css

```
@@ -455,17 +482,269 @@ governing permissions and limitations under the License.
 /* backwards compatibility: these are not required, but they make the handle go to the edegs and align right */
 .spectrum-Slider-handleContainer,
 .spectrum-Slider-trackContainer {
-  width: calc(100% + var(--spectrum-slider-handle-width));
+  width: calc(100% + var(--spectrum-slider-handle-size));
 
   position: absolute;
-  top: calc(var(--spectrum-slider-track-height) / 2 - 1px);
+  top: calc(var(--mod-slider-control-height, var(--spectrum-slider-control-height)) / 2 - 1px);
 
-  margin-left: calc(var(--spectrum-slider-handle-width) / 2 * -1);
+  margin-left: calc(var(--spectrum-slider-handle-size) / 2 * -1);
 }
 
 .spectrum-Slider-trackContainer {
-  height: var(--spectrum-slider-height);
+  height: var(--mod-slider-height-size, var(--spectrum-slider-height-size));
```

```suggestion
  block-size: var(--mod-slider-height-size, var(--spectrum-slider-height-size));
```
Let's use the logical property here for some future-proofing.